Scripts/git-hooks/pre-commit

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# Run swift format on the Sources directory
+echo "Running Swift format..."
+swift format --recursive --in-place Sources/
+
+# Add formatted files back to the commit
+git add Sources/
```

Comment:
Wouldn't this prevent you from committing only some source files?

I personally just use the Format on Save feature in VS Code

Reply:
here's another example of why I want this:

https://github.com/edgeengineer/edge-agent/pull/31 

I keep seeing them here and there.

Reply:
I don't think we need to `git add Sources/` but we can still have the formatting without that

Reply:
At least it'll show you that you missed those, so you can quickly amend them